app/controllers/admin/tasks_controller.rb

```diff
@@ -27,6 +27,8 @@ def show
       @claim.notes.order(created_at: :desc).select { |note| note.body.include?("ligible") && note.body.include?("Subject 1") }
     elsif params[:name] == "employment"
       @claim.notes.order(created_at: :desc).select { |note| note.body.include?("ligible") && note.body.include?("[Employment]") }
+    elsif params[:name] == "induction_confirmation"
+      @claim.notes.order(created_at: :desc).select { |note| note.body.include?("ligible") && note.body.include?("Completion date") }.uniq { |n| n.body }
```

Comment:
@slorek I agree; this is pretty horrible. The only "clean" way I can think of is to link `notes` with `tasks` with a relationship. That way, we could retrieve all notes for a particular task (by name). However, this would require a data migration for the existing notes and tasks. Otherwise, we can simply move this lookup logic to the models (using scopes, which might be more readable and efficient). Any thoughts?